apps/angular/src/app/features/anime/animes-page/animes-page.component.html

```diff
@@ -1,12 +1,28 @@
 <div class="animes">
+	<form [formGroup]="form" (ngSubmit)="onSubmit()" class="animes__form form">
+		<mat-form-field color="accent" class="form__field">
```

Comment:
Looks like you are not using it
```suggestion
		<mat-form-field color="accent">
```